Cargo.toml

```diff
@@ -71,14 +73,15 @@ path = "./askar-crypto"
 features = ["all_keys", "any_key", "argon2", "crypto_box", "std"]
 
 [dependencies.sqlx]
-version = "0.5.13"
+version = "0.6.2"
 default-features = false
 features = ["chrono", "runtime-tokio-rustls"]
 optional = true
 
 [profile.release]
-lto = true
 codegen-units = 1
+lto = true
+panic = "abort"
```

Comment:
What's the reason behind abort?

Reply:
It removes the panic unwinding, because a panic can't cross the FFI boundary and it potentially leaves things in an invalid state. This makes the binary smaller as well.